docs/docs/build/presence.md

```diff
@@ -110,6 +110,42 @@ Notifications API is partially implemented. All messages are always broadcast ev
 
 Notifications are fundamentally unreliable at this time as there are no built-in acknowledgements nor retained state. To prevent most common loss of notifications, always check for connection before sending.
 
-### Throttling
+### Throttling/grouping
 
-Throttling is not yet implemented. `BroadcastControls` exists in the API to provide control over throttling of value updates, but throttling is not yet implemented. It is recommended that `BroadcastControls.allowableUpdateLatencyMs` use is considered and specified to light up once support is added.
+Presence updates are grouped together and throttled to prevent flooding the network with messages when presence values are rapidly updated. This means the presence infrastructure will not immediately broadcast updates but will broadcast them after a configurable delay.
+
+The `allowableUpdateLatencyMs` property configures how long a local update may be delayed under normal circumstances, enabling grouping with other updates. The default `allowableUpdateLatencyMs` is **60 milliseconds** but may be (1) specified during configuration of a [States Workspace](#states-workspace) or [Value Manager](#value-managers) and/or (2) updated later using the `controls` member of Workspace or Value Manager. [States Workspace](#states-workspace) configuration applies when a Value Manager does not have its own setting.
+
+Notifications are never queued; they effectively always have an `allowableUpdateLatencyMs` of 0. However, they may be grouped with other updates that were already queued.
```

Comment:
I remember Kash asking why notifications are not queued during standup. Maybe a small sentence explaining why we allow these to skip the line.

Reply:
Technically (and actually important), they don't skip the line. They do cause a flush. So everything currently queued also goes out. Let's customers see rational ordering if they want to set some state and then do a notification.
This is a good note, but it might be too much for the customer facing documentation. We can say what it does without the details. If you were reading this as customer, do you think it is important to know why this behavior is that way? ...or just that that is the way it is?

Reply:
I think as a customer I'd like to know a notification causes a flush! But I agree that maybe going into why might not be too important for customer facing docs.

Reply:
I thought about it some and I think the wording is enough to indicate that is what happens without making it a written contract requirement. I can't think of reason we'd not do that, but I want to leave the door open for the moment.